modules/importing-manifest-list-through-imagestreamimport.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+// * assembly/openshift_images/managing-image-streams.adoc
+
+:_content-type: PROCEDURE
+[id="importing-manifest-list-through-imagestreamimport_{context}"]
+= Importing a manifest list through ImageStreamImport
+
+Use the following procedure to import a manifest list through the `ImageStreamImport` object with the `importMode` value. 
+
+.Procedure
+
+. Create an `ImageStreamImport` object that sets the `importMode` to `Manifestlist` by running the following command:
++
+[source,terminal]
+----
+$ cat <<EOF | oc create -f -
+apiVersion: image.openshift.io/v1
+kind: ImageStreamImport
+metadata:
+  name: app
+  namespace: <namespace>
+spec:
+  import: true
+  images:
+  - from:
+      kind: DockerImage
+      name: quay.io/<user_name>/<image_name>
+    to:
+      name: latest
+    referencePolicy:
+      type: Source
+    importPolicy:
+      importMode: "ManifestList"
+EOF
+----
+
+. Enter the following command to ensure that Image objects were created for the manifest list: 
++
+[source,terminal]
+----
+$ oc get image | grep quay.io/<user_name>/<image_name>
+----
+
+. Enter the following command to ensure that all sub-manifests were created:
++
+[source,terminal]
+----
+$ oc get is app
+----
++
+[NOTE]
+====
+You can compare the output from the aforementioned commands against the content of `quay.io/<user_name>/<image_name>`. If correct, there should be one image for the manifest list, and one image for each sub-manifest. 
```

Comment:
I'm unsure about the usefulness of this note to the end-user... Shouldn't they be able to trust that the image stream was created correctly without the need to verify it? Perhaps this is better as a troubleshooting step? 🤔 
I mostly added it to the [step-by-step](https://github.com/flavianmissi/imagestreams/blob/main/manifest-list-import.md) to aid testing.